samples/dotnet/EdgeX509AuthDownstreamDevice/Program.cs

```diff
@@ -218,17 +239,19 @@ static void Main()
                 }
             }
 
-            TransportType transport = GetTransport(ClientTransportType);
-
             InstallCACert();
 
+            X509Certificate2 trustedCACert = getTrustedCACertFromFile(TrustedCACertPath);
```

Comment:
This does not need to be at this high of scope level since TrustedCACertPath is global. I will move it into the implementation of GetTransport().